api/v1alpha1/etcdpeer_types.go

```diff
@@ -27,12 +27,26 @@ type StaticBootstrap struct {
 	InitialCluster []InitialClusterMember `json:"initialCluster,omitempty"`
 }
 
+const InitialClusterStateNew InitialClusterState = "New"
+const InitialClusterStateExisting InitialClusterState = "Existing"
```

Comment:
Consider making these lower case to match the values expected by the etcd `--initial-cluster-state` argument

Reply:
I made them leading upper case to match other cases where strings are used as enum values in the Kubernetes API. For example a contaner's `imagePullPolicy` can be `Always`, `Never`, `IfNotPresent`.